### PR TITLE
Testing for new refactorying style

### PR DESCRIPTION
**Pull request by hansenchen66**

### CODE REVIEW

**Comment by hansenchen66** [comment]
fixes #112 #111 #110 #109

File: src/Main.java [hunk 1]
```diff
@@ -36,7 +38,10 @@
 	 */
 	public static void main(String[] args) {
 		Warehouse warehouse = new Warehouse();
-		EntityCommand[] commands = new EntityCommand[] { new OrderCommand(warehouse), new PickerCommand(warehouse) };
+		Server server = new Server(warehouse);
+		FaxMachine faxmachine = new FaxMachine(server);
+		warehouse.AddFaxMachine(faxmachine);
+		EntityCommand[] commands = new EntityCommand[] { new OrderCommand(faxmachine), new PickerCommand(warehouse) };
```
Comment (PkHutch) [inline src/Main.java]: Over line limit.

File: src/Main.java [hunk 2]
```diff
@@ -97,7 +102,7 @@ private static void saveFinalState(Floor floor) throws IOException {
 		// level.
 		// Then write the resulting new line to the file if the line exists.
 		System.out.println("The current state of warehouse has been saved");
```
Comment (PkHutch) [inline src/Main.java]: The Warehouse hasn't been saved, the Warehouse is beginning the process of being saved.

**Review by PkHutch — CHANGES_REQUESTED** [review]

File: src/entities/Level.java [hunk 3]
```diff
@@ -53,4 +63,31 @@ public String getLocation() {
...
+	
+	public void addStock(){
+		this.stock += 1;
+	}
+	
+	public Integer getSku() {
+		return sku;
+	}
+	
+	public void setSku(Integer sku) {
```
Comment (PkHutch) [inline src/entities/Level.java]: This should not exist, the SKU never changes.

**Review by PkHutch — CHANGES_REQUESTED** [review]

**Pull request closed without merging**